resource/js/vocab-search.js

```diff
@@ -6,49 +6,246 @@ const vocabSearch = Vue.createApp({
       languages: [],
       selectedLanguage: null,
       searchTerm: null,
-      autoCompeteResults: [],
-      languageStrings: null
+      searchCounter: null,
+      renderedResultsList: [],
+      languageStrings: null,
+      msgs: null
```

Comment:
I would add a variable to control the visibility of the dropdown list instead of manipulating DOM directly
```suggestion
      msgs: null
      showDropdown: false
```